src/libsyntax/parse/parser/diagnostics.rs

```diff
@@ -1092,7 +1092,7 @@ impl<'a> Parser<'a> {
             Err(mut err) => {
                 err.emit();
                 // Recover from parse error.
-                self.consume_block(delim);
+                self.consume_block(delim, true);
```

Comment:
The `true` here wants a comment as to the "why".

Reply:
@estebank I believe this comment was not addressed.

Reply:
Let me know if the new comment is enough.